src/ucp/core/ucp_worker.c

```diff
@@ -2024,6 +2029,7 @@ void ucp_worker_destroy(ucp_worker_h worker)
     ucp_worker_wakeup_cleanup(worker);
     ucs_mpool_cleanup(&worker->rkey_mp, 1);
     ucs_mpool_cleanup(&worker->req_mp, 1);
+    ucp_worker_keepalive_deactivate(worker);
```

Comment:
why needed? removing all eps should deactivate the keepalive

Reply:
ok, removed